pkg/scheduler/framework/plugins/noderesources/fit.go

```diff
@@ -174,11 +178,11 @@ type InsufficientResource struct {
 }
 
 // Fits checks if node have enough resources to host the pod.
-func Fits(pod *v1.Pod, nodeInfo *schedulernodeinfo.NodeInfo, ignoredExtendedResources sets.String) []InsufficientResource {
-	return fitsRequest(computePodResourceRequest(pod), nodeInfo, ignoredExtendedResources)
+func Fits(pod *v1.Pod, nodeInfo *schedulernodeinfo.NodeInfo, ignoredExtendedResources, ignoreResourceGroups sets.String) []InsufficientResource {
```

Comment:
It looks like no users of `Fits` care about ignored resources. Could you just remove both?

Reply:
done